templates/taxi/driver_list.html

```diff
@@ -5,13 +5,25 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
+          {% if driver.username == user.username  %}
+            <li>
+              <a href="{% url "taxi:driver_detail" pk=driver.id %}">
+                  {{ driver.username }}
+              </a>(Me)
+            </li>
+          {% else %}
+              <li>
+              <a href="{% url "taxi:driver_detail" pk=driver.id %}">
+                  {{ driver.username }}
+              </a>
```

Comment:
don't repeat yourself: add username and only then check if me or first/last names should be on the page